vignettes/E_Genomics.Rmd

```diff
@@ -383,22 +378,47 @@ qI <- getQueen(colony)@id
 fI <- sort(getFathers(colony)@id)
 wI <- sort(getWorkers(colony)@id)
 dI <- sort(getDrones(colony)@id)
+r <- range(GRMs$indiv)
 ```
 
-Compare queen and fathers.,
+Compare queen and fathers:
 
 ```{r  Queen vs fathers}
-head(GRMs$indiv[fI, qI])
+hist(GRMs$indiv[fI, qI], xlim = r)
 ```
 
-queen and workers,
+Queen and workers:
 
 ```{r  Queen vs workers}
-head(GRMs$indiv[wI, qI])
+hist(GRMs$indiv[wI, qI], xlim = r)
 ```
 
-and queen and drones,.
+Queen and drones:
 
 ```{r  Queen vs drones}
-head(GRMs$indiv[dI, qI])
+hist(GRMs$indiv[dI, qI], xlim = r)
 ```
+
```

Comment:
Can you add a title "References" here? We have it in other vignettes as well

Reply:
Good point. Thanks for spotting this!